### ISSUE TITLE
The `.conda` directory is not being created on Windows

### ISSUE THREAD

**Issue by jlstevens** [conda/constructor#461]
When trying to build a Windows installer (win-64, tested on Windows 10), it was found that the uninstaller wasn't removing menu entries but only in some Windows environments and not others. After some investigation, it turned out that the environments where the uninstaller *does* work properly are the ones where a `.conda` directory already existed due to a previous Miniconda/Anaconda installation.

The problem seems to be that the `.conda` directory (usually in `$HOME`) is not created by the installers built by constructor. In turn, it seems that the issue with the uninstaller (i.e. not removing menu items) goes away when the available environments can be found in the `environments.txt` file of the `.conda` directory (when present).

In conclusion, constructor built installers do not appear to be creating the `.conda` directory when they should while the vanilla Miniconda/Anaconda installers have no such issue. I've discussed this with @chenghlee and the root cause of this discrepancy is currently being investigated.

**Comment by jaimergp** [comment]
Some notes from my investigation:

The uninstaller is built with these instructions:

https://github.com/conda/constructor/blob/7a67384f92e9c5b628a02d91a81e9d43bf0c5412/constructor/nsis/main.nsi.tmpl#L972-L1004

The menu removal happens through the bundled `conda-standalone` executable, which itself bundles the `_nsis.py` script in `constructor` - this is the function that gets called with the `--rm-menus` option:

https://github.com/conda/constructor/blob/7e76a572d5b87b3aa319218fd73c3cc48506b6e3/constructor/nsis/_nsis.py#L159-L184

We can clearly see that a non-existing `.conda/environments.txt` will make this "just-in-case" error checking.

Why do we need this environments check? I guess because `constructor` was designed as a `conda` installer, so the user might have created more environments using it. However, some people use `constructors` to distribute installers _without_ conda, so this makes not sense in that use case. We should have a fallback mode there that doesn't assume `.conda` exists. In those cases, the installer probably didn't have `conda` bundled (or was never executed), which means we can just clear the passed `prefix` normally.

PR coming soon.
